wiki/Gameplay/Drain_time/en.md

```diff
@@ -6,8 +6,8 @@ tags:
 
 # Drain time
 
-**Drain time** refers to the total duration of a [beatmap](/wiki/Beatmap) that requires the player to be active. It begins at the first [hit object](/wiki/Hit_object), ends at the end of the last hit object, and does not include breaks. The word *drain* originates from [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch)'s [health drain](/wiki/Beatmapping/Health_drain) system, where the player's health is constantly decreasing as they play the map.
+**Drain time** refers to the total duration of a [beatmap](/wiki/Beatmap) that requires the player to be active. It begins at the first [hit object](/wiki/Hit_object), ends at the end of the last hit object, and does not include [breaks](/wiki/Beatmap/Break). The word *drain* originates from [osu!](/wiki/Game_mode/osu!) and [osu!catch](/wiki/Game_mode/osu!catch)'s [health drain](/wiki/Beatmapping/Health_drain) system, where the player's health is constantly decreasing as they play the map.
 
 Beatmaps with very long drain times are typically referred to as *[marathons](/wiki/Beatmap/Marathon)* due to requiring endurance beyond the norm in order to be completed.
 
-The beatmap info page on the website<!-- TODO: link --> calculates drain time (labelled `Drain length`) based on only hit objects' start times, so if the last object of the map is a [slider](/wiki/Hit_object/Slider) or [spinner](/wiki/Hit_object/Spinner), the drain time will be displayed incorrectly.
+The beatmap info page on the [website](https://osu.ppy.sh) calculates drain time (labelled `Drain length`) based on only hit objects' start times, so if the last object of the map is a [slider](/wiki/Hit_object/Slider) or [spinner](/wiki/Hit_object/Spinner), the drain time will be displayed incorrectly.
```

Comment:
I sort of disagree with this todo, since the "website" is mentioned in a lot more places and this would only make sense in the lazer client (and even then would be even more irrelevant if wiki articles on lazer get button that allows one to view the page on the site anyway). On the other hand, you're already reading this on the website.

Pinging @cl8n since github blame points at you

```suggestion
The beatmap info page on the website calculates drain time (labelled `Drain length`) based on only hit objects' start times, so if the last object of the map is a [slider](/wiki/Hit_object/Slider) or [spinner](/wiki/Hit_object/Spinner), the drain time will be displayed incorrectly.
```

Reply:
I think this was just me going trigger happy with "TODO: link", not sure what I thought it should link to...

Reply:
the beatmap info page, perhaps? we have https://github.com/ppy/osu-wiki/issues/3122

Reply:
seems most likely, sorry I can't remember